### ISSUE TITLE
Fatal error in env:setup

### ISSUE THREAD

**Issue by tellyworth** [WordPress/wporg-news-2021#89]
There's a new fatal error that's happening on fresh installs:

```
% yarn run env:setup
yarn run v1.22.10
$ wp-env start --update && yarn run env:reset && yarn run env:import && yarn run env:misc
✖ Error while running docker-compose command.
WordPress is already installed.
Success: Updated the constant 'WP_DEBUG' in the 'wp-config.php' file with the raw value 'true'.
Success: Updated the constant 'SCRIPT_DEBUG' in the 'wp-config.php' file with the raw value 'true'.
Success: Updated the constant 'WP_ENVIRONMENT_TYPE' in the 'wp-config.php' file with the value 'local'.
Success: Updated the constant 'WP_PHP_BINARY' in the 'wp-config.php' file with the value 'php'.
Success: Updated the constant 'WP_TESTS_EMAIL' in the 'wp-config.php' file with the value 'admin@example.org'.
Success: Updated the constant 'WP_TESTS_TITLE' in the 'wp-config.php' file with the value 'Test Blog'.
Success: Updated the constant 'WP_TESTS_DOMAIN' in the 'wp-config.php' file with the value 'http://localhost:8888/'.
Success: Updated the constant 'WP_SITEURL' in the 'wp-config.php' file with the value 'http://localhost:8888/'.
Success: Updated the constant 'WP_HOME' in the 'wp-config.php' file with the value 'http://localhost:8888/'.
Success: Updated the constant 'WP_DEBUG_LOG' in the 'wp-config.php' file with the value '/tmp/wp-errors.log'.
Success: Updated the constant 'FS_METHOD' in the 'wp-config.php' file with the value 'direct'.
Success: Updated the constant 'JETPACK_DEV_DEBUG' in the 'wp-config.php' file with the raw value 'true'.
Success: Updated the constant 'WPORG_SANDBOXED' in the 'wp-config.php' file with the raw value 'true'.
Creating d9c62a1976b7f8e148989554512bd4ca_cli_run ... 
Creating d9c62a1976b7f8e148989554512bd4ca_cli_run ... done
Warning: filemtime(): stat failed for /var/www/html/wp-content/mu-plugins/wporg-mu-plugins/mu-plugins/blocks/global-header-footer/build/style.css in /var/www/html/wp-content/mu-plugins/wporg-mu-plugins/mu-plugins/blocks/global-header-footer/blocks.php on line 20
Warning: filemtime(): stat failed for /var/www/html/wp-content/mu-plugins/wporg-mu-plugins/mu-plugins/blocks/global-header-footer/build/style.css in /var/www/html/wp-content/mu-plugins/wporg-mu-plugins/mu-plugins/blocks/global-header-footer/blocks.php on line 20
Fatal error: Cannot declare interface WP_Theme_JSON_Schema, because the name is already in use in /var/www/html/wp-content/plugins/gutenberg/lib/interface-wp-theme-json-schema.php on line 13
255
error Command failed with exit code 255.
```

I was able to work around it by changing `.wp-env.json` to use the core #5.8-branch instead of #master. Not sure how to fix the underlying issue.

**Comment by iandunn** [comment]
I think that's just a temporary issue, similar to https://core.trac.wordpress.org/ticket/54401 / https://core.trac.wordpress.org/ticket/54336. I'm guessing it'll go away once Core `trunk` and Gutenberg `trunk` are in sync.

Also, Core's mirror changed from `master` to `trunk` recently, so it may just be that they're already back in sync, but `master` isn't getting new commits.

So, we should probably try switching to `WordPress/WordPress#trunk` and re-provisioning. Not a high priority, though, probably after SotW is done. Waiting a few weeks will also give any Core/Gutenberg sync issues a chance to resolve.

**Comment by ryelle** [comment]
I think it's actually because the composer.lock file is outdated, so the Gutenberg version installed when running the setup (11.7 IIRC) conflicts with trunk.

The branch used in `.wp-env.json` should map to the branch in [this repo, WordPress/WordPress](https://github.com/WordPress/WordPress/), not the wordpress-develop repo, and the branch name hasn't changed there. So `"core": "WordPress/WordPress#master",` is still correct.

After I ran `composer update` I was able to switch to `WordPress/WordPress#master` without issue.
